include/envoy/upstream/cluster_manager.h

```diff
@@ -315,6 +317,8 @@ class ClusterInfoFactory {
     const LocalInfo::LocalInfo& local_info_;
     Event::Dispatcher& dispatcher_;
     Runtime::RandomGenerator& random_;
+    Singleton::Manager& singleton_manager_;
+    ThreadLocal::SlotAllocator& tls_;
```

Comment:
nit: s/tls/thread_local here and in the rest of the change?

Reply:
`thread_local` is a reserved word. I will leave it as is.

Reply:
Okay.